dev-tex/rubber/rubber-1.6.0_alpha1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS="no"
+
+inherit distutils-r1
+
+if [[ ${PV} == "9999" ]] || [[ -n "${EGIT_COMMIT_ID}" ]]; then
```

Comment:
This logic is only worthwhile if we're keeping a live ebuild too (to allow them to both be kept in sync). But there isn't one right now.